src/neuroconv/datainterfaces/behavior/video/videodatainterface.py

```diff
@@ -28,7 +28,12 @@ class VideoInterface(BaseDataInterface):
     # Other suffixes, while they can be opened by OpenCV, are not supported by DANDI so should probably not list here
     info = "Interface for handling standard video file formats."
 
-    def __init__(self, file_paths: list, verbose: bool = False):  # TODO - debug why List[FilePathType] fails
+    def __init__(
+        self,
+        file_paths: list,
```

Comment:
Maybe a good point to start with keywords only on new arguments that we add?

Reply:
@h-mayorquin I'm but sure I understand what you mean